src/zeit/content/article/edit/interfaces.py

```diff
@@ -427,3 +430,53 @@ class IBreakingNewsBody(zope.interface.Interface):
 
     article_id = zope.interface.Attribute(
         'The uniqueID of the breaking news article')
+
+
+class Puzzle(zeit.cms.content.sources.AllowedBase):
+
+    def __init__(self, id, title, multiple):
+        super(Puzzle, self).__init__(id, title, None)
+        self.multiple = multiple
+
+    def __eq__(self, other):
+        return super(Puzzle, self).__eq__(
+            zope.security.proxy.removeSecurityProxy(other))
```

Comment:
Also an [anderen Stellen](https://github.com/ZeitOnline/zeit.cms/blob/master/src/zeit/cms/content/configure.zcml#L112) machen wir da tatsächlich eine Security-Declaration. Das ist zwar bürokratisch, ich würd aber removeProxy gern nur in Ausnahmefällen (und mit Kommentar wieso weshalb warum) verwenden.